Fix #267, explain why we are not using JSON Path or JSON Pointer (#295)
oauth-wg/oauth-sd-jwt-vc: draft-ietf-oauth-sd-jwt-vc.md

consuming application which does not have access to all disclosures may not be
able to identify the claim which is being addressed.

> Note: This specification does intentionally not use JSON Pointer [@RFC6901] for

Reviewer: more natural phrasing to a native English speaker 
```suggestion
Note: This specification intentionally does not use JSON Pointer [@RFC6901] for
```

> Note: This specification intentionally does not use JSON Pointer [@RFC6901] for

Reviewer: let's be explicit about the "?"ness 
```suggestion
Note: This specification intentionally does not use JSON Pointer [@?RFC6901] for
```

selecting claims, as JSON Pointer requires string parsing and does not support
wildcard selection of array elements. It does not use JSON Path [@?I-D.goessner-dispatch-jsonpath] as

> that introduces a considerable complexity and brings in a lot of features that
> are not needed for the use case of selecting claims in a credential. There are
> also security concerns with some implementations.

Reviewer 2: editorial nit

```suggestion
that introduces a considerable complexity and brings in many features which
are not needed for the use case of selecting claims in a credential. There are
also security concerns with some implementations.
```
